ci/scripts/run-tests-linux.sh

```diff
@@ -27,6 +27,7 @@ function run_db_tests(){
   then
     cd $WORKDIR/build && \
     make test && \
+    make test-client
```

Comment:
If it is trivial to run this on mac as well, let's do that. But otherwise it is fine as is.

Reply:
Added to mac tests as well.